crates/bevy_ui/src/flex/mod.rs

```diff
@@ -30,16 +30,16 @@ unsafe impl Sync for FlexSurface {}
 
 fn _assert_send_sync_flex_surface_impl_safe() {
     fn _assert_send_sync<T: Send + Sync>() {}
-    _assert_send_sync::<HashMap<Entity, stretch::node::Node>>();
-    _assert_send_sync::<HashMap<WindowId, stretch::node::Node>>();
+    _assert_send_sync::<HashMap<Entity, taffy::node::Node>>();
+    _assert_send_sync::<HashMap<WindowId, taffy::node::Node>>();
     // FIXME https://github.com/vislyhq/stretch/issues/69
```

Comment:
@alice-i-cecile the FIXME link is fixed on the latest commit. I'm unsure why github hasn't marked that as outdated?